app/bin/tools/check_drift.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:math' as math;
+
+import 'package:http/http.dart';
+import 'package:pub_dev/search/search_form.dart';
+import 'package:pub_dev/shared/tags.dart';
+
+final _client = Client();
+final _random = math.Random.secure();
+
+final _forms = <SearchForm>[
+  SearchForm.parse(
+    tagsPredicate: TagsPredicate(requiredTags: [PackageTags.isFlutterFavorite]),
+  ),
+  SearchForm.parse(query: 'json'),
+  SearchForm.parse(sdk: 'dart', currentPage: 2),
+  SearchForm.parse(sdk: 'flutter', currentPage: 5),
+  SearchForm.parse(order: SearchOrder.created),
+  SearchForm.parse(order: SearchOrder.updated, currentPage: 3),
+];
+
+Future<void> main() async {
```

Comment:
A `--help` option wouldn't hurt... and a few words on what this does.. Notably, it worth saying that it WON'T use any form of administrative access or modify server-side resources. It only does a series of `GET` requests, right?